tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -90,7 +223,10 @@ type SyntheticProject =
       SourceFiles: SyntheticSourceFile list
       DependsOn: SyntheticProject list
       RecursiveNamespace: bool
-      OtherOptions: string list }
+      OtherOptions: string list
+      AutoAddModules: bool
+      NugetReferences: (string * string option) list
```

Comment:
Some thin type wrappers would be nice, to make it more obvious what " string * string option" is supposed to be.